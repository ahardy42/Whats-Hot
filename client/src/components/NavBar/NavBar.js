import React, {useState} from 'react';
import './NavBar.sass';

const NavBar = (props) => {
    return(
        <nav className="navbar">
            <span className="navbar-brand">Hey!</span>
        </nav>
    )
}

export default NavBar;