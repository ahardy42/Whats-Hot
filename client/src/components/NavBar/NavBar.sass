@import ../../../colorVars
    
.myNav
    height: 100vh
    width: 100vw
    background-color: $awesomeGreen
    color: $awesomeOrange
    z-index: 0
    text-align: center
    a
        position: relative
        top: 2.5vh