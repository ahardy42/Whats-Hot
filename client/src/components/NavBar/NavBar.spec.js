import React from 'react';
import NavBar from './NavBar';
import renderer from 'react-test-renderer';

describe('navbar render test', () => {
    it('renders the navbar in landing page view', () => {
        const tree = renderer.create(
            <NavBar isMapView={false}/>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})