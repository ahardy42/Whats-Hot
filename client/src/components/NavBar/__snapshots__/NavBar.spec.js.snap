// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navbar render test renders the navbar in landing page view 1`] = `
<nav
  className="myNav"
/>
`;
